Public/Src/Sandbox/MacOs/Sandbox/Src/Resources/Info.plist

```diff
@@ -7,7 +7,7 @@
 	<key>CFBundleExecutable</key>
 	<string>$(EXECUTABLE_NAME)</string>
 	<key>CFBundleIdentifier</key>
-	<string>$(BUILDXL_BUNDLE_IDENTIFIER)</string>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
```

Comment:
>PRODUCT_BUNDLE_IDENTIFIER [](start = 11, length = 25)

didn't see corresponding define change...

Reply:
`PRODUCT_BUNDLE_IDENTIFIER` is the default entry for plist files, we assign the our custom identifier value to it inside the Sandbox project:

```PRODUCT_BUNDLE_IDENTIFIER = "$(BUILDXL_BUNDLE_IDENTIFIER)";```